CyberConnect2/nucc.bt

```diff
@@ -133,48 +134,52 @@ typedef struct
 
 typedef struct
 {
-    u16 Sec1Count;
-    u16 Sec2Count;
+    u16 SpringGroupsCount;
+    u16 CollisionSpheresCount;
 
     u32 ClumpChunkIndex;
 
-    local int i, j, childrenCount = 0;
-    for ( i = 0; i < Sec1Count; i++ )
+    local int i, j, bonesCount = 0;
+    for ( i = 0; i < SpringGroupsCount; i++ )
     {
         struct
         {
-            f32 Bounce;
-            f32 Speed;
-            f32 Gravity;
-            f32 Influence;
-
+            f32 Bounciness;
+            f32 Elasticity;
+            f32 Stiffness;
+            f32 Movement;
             u16 CoordIndex;
-            u16 ChildCount;
-            childrenCount += ChildCount;
-        } Sec1;
+            u16 BonesCount;
+            bonesCount += BonesCount;
+        } SpringGroup;
     }
 
-    for ( i = 0; i < Sec2Count; i++ )
+    for ( i = 0; i < CollisionSpheresCount; i++ )
     {
         struct
         {
-            f32 Values[6];
+            f32 OffsetZ;
+            f32 OffsetY;
+            f32 OffsetX;
+            f32 ScaleZ;
+            f32 ScaleY;
+            f32 ScaleX;
```

Comment:
Are these coordinates actually reversed, or is this based on Blender's coordinate system?

The template should be in the game's coordinate system, so these should be XYZ unless they're actually ZYX in the game as well.

Reply:
Not sure about this one. These were the changes Hydra made so I assume they were correct. Before using Blender he would use 010 editor to edit the values and it seems he figured they must be flipped.